src/transport_picoquic.h

```diff
@@ -165,13 +165,19 @@ class PicoQuicTransport : public ITransport
     /*
    * Internal Public Variables
      */
-    LogHandler& logger;
+    cantina::LoggerPointer logger;
+    int logging_fds[2];
```

Comment:
Maybe add a comment describing where/why this is used.

Reply:
I'm most definitely not opposed to comments, but there aren't others here. Personally, I like a simple comment on every single variable, but most have nothing. I didn't want to create inconsistencies.